cmd/e2equeries/main.go

```diff
@@ -44,7 +44,9 @@ func main() {
 	rowchan, _ := db.Transactions(context.Background(), rekeyTxnQuery)
 	for txnrow := range rowchan {
 		maybeFail(txnrow.Error, "err rekey txn %v\n", txnrow.Error)
-		rekeyedAuthAddrs = append(rekeyedAuthAddrs, txnrow.Txn.Txn.RekeyTo)
+		if txnrow.Txn != nil {
+			rekeyedAuthAddrs = append(rekeyedAuthAddrs, txnrow.Txn.Txn.RekeyTo)
+		}
```

Comment:
I'm not exactly sure whats happening here, but perhaps we need to check both
```suggestion
                t := txnrow.Txn
		if txnrow.RootTxn != nil {
		        t = txnrow.RootTxn
		}
		rekeyedAuthAddrs = append(rekeyedAuthAddrs, t.Txn.RekeyTo)
```